modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-webengine/src/main/resources/skin/resources/script/jquery.toc.js

```diff
@@ -0,0 +1,25 @@
+(function ($) {
+
+  $.fn.toc = function (tocListContainer, tocList) {
+    $(tocList).empty();
+
+    var index = 0;
+    $("h2").each(function () {
+      var name = "toc" + index;
+      var anchor = "<a name='" + name + "' class='anchor'></a>";
```

Comment:
Use String templating for that: https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/template_strings

Reply:
i changed the logics, you'll tell me if you prefer or not